### ISSUE TITLE
ZXing and frame rotation

### ISSUE THREAD

**Issue by daankennes** [xulihang/vision-camera-zxing#2]
If the frame in the frameProcessor has a rotation (frame.orientation equals landscape-right, landscape-left or portrait-upside-down), codes will not scan because ZXing requires the the code to be upright. Is there a work-around for this?

**Comment by xulihang** [comment]
QR code can be read in any direction while 1D code cannot. I didn't find a solution.

**Comment by daankennes** [comment]
That makes this module not very usable, or am I overlooking something? 

- I tried rotating the frame with vision-camera-resize-plugin, but I end up with a uint8Array which I cannot pass to the decoder function.
- I tried rotating the frame with vision-camera-resize-plugin, converting the result to base64 and feeding that to the base64 decoder function. That is a) not possible because frame processors are synchronous and would b) be very slow even if it worked.
